### PR TITLE
[INFINITY-2149] Ensure all plugins get installed successfully

### PR DESCRIPTION
**Pull request by loren**
- If a plugin fails to install, exit the script immediately with an error so the task fails and can be restarted

### CODE REVIEW

**Comment by loren** [comment]
I fixed a problem, but not the one causing the errors. Going to fix that, too, and then reopen PR.

File: frameworks/elastic/tests/test_sanity.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import logging
```
Comment (NimaVaziri) [inline frameworks/elastic/tests/test_sanity.py]: Was this unintentional? 

`logging` is being used on L12

Comment (NimaVaziri) [inline frameworks/elastic/tests/test_sanity.py]: Oh importing everything from `tests.config` includes an import of `logging`

That's not immediately clear, hence we should do a cleanup in a separate PR.

Comment (loren) [inline frameworks/elastic/tests/test_sanity.py]: Yes, I saw another PR (I forget whose) that tried to remove `from tests.config import *` from all the shakedown files. Not sure what became of it.
